storage/innobase/row/row0ins.cc

```diff
@@ -2774,6 +2772,29 @@ row_ins_clust_index_entry_low(
 			export_vars.innodb_bulk_operations++;
 			goto err_exit;
 		}
+	} else if (flags == (BTR_NO_UNDO_LOG_FLAG | BTR_NO_LOCKING_FLAG)
+		   && !index->table->has_spatial_index()
+		   && !index->table->versioned()
+		   && !index->table->is_temporary()
+		   && !index->table->n_rec_locks) {
+
+		ut_ad(index->table->skip_alter_undo);
+		ut_ad(!entry->is_metadata());
+		if (innodb_alter_copy_bulk) {
+			ut_ad(page_is_empty(block->page.frame));
+			/* This code path has been executed at the
+			start of the alter operation. Consecutive
+			insert operation are buffered in the
+			bulk buffer and doesn't check for constraint
+			validity of foreign key relationship. */
+			trx_start_if_not_started(trx, true);
+			trx->bulk_insert = true;
+			auto m= trx->mod_tables.emplace(index->table, 0);
+			m.first->second.start_bulk_insert(index->table);
+			err= m.first->second.bulk_insert_buffered(
+						*entry, *index, trx);
```

Comment:
Here, we are sometimes omitting a space before `=`, sometimes adding it. In the archaic InnoDB formatting style, which this function is using, there should be a space on both sides of `=`.

Only the archaic InnoDB formatting style allows a line break after `(`. The following line should be indented exactly 8 columns (1 TAB) deeper per unbalanced `(`. Here the line is indented too deep.